interview-questions.md

```diff
@@ -8,50 +8,123 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
When answering these interview questions, please remember to use your own words. Do the research then paraphrase. I love how you added personality and showed examples from class. Next week, let's focus on complete sentences that address the definition of the topic. Including an example either from class or you personal experience.